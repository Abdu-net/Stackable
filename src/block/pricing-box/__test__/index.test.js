import { edit, save } from '../index'

describe( 'Pricing Box', () => {
	test( 'block edit matches snapshots', () => {
		const wrapper = edit( {
			isSelected: false,
			attributes: {
				url: 'http://www.gambit.ph/',
				pricingBoxTitle: 'Basic',
				price: '$9',
				perMonthLabel: 'per month',
				buttonText: 'Buy Now',
				featureList: 'Consectetur adipiscing elit Suspendisse at pretium tortor Vestibulum ante ipsum primis In faucibus orci luctus et Ultrices posuere cubilia cura Aenean consectetur nec',
				pricingBoxColor: '#000000',
				priceColor: '#000000',
				perMonthLabelColor: '#000000',
				buttonColor: '#000000',
				buttonTextColor: '#000000',
				featureListColor: '#000000',
				columns: '1',
				size: 'normal',
				cornerButtonRadius: '4',
			},

		} )

		expect( wrapper ).toMatchSnapshot()
	} )

	test( '2 column block edit matches snapshots', () => {
		const wrapper = edit( {
			isSelected: false,
			attributes: {
				url: 'http://www.gambit.ph/',
				url2: 'http://www.gambit.ph/',
				pricingBoxTitle: 'Basic',
				pricingBoxTitle2: 'Basic',
				price: '$9',
				price2: '$9',
				perMonthLabel: 'per month',
				perMonthLabel2: 'per month',
				buttonText: 'Buy Now',
				buttonText2: 'Buy Now',
				featureList: 'Consectetur adipiscing elit Suspendisse at pretium tortor Vestibulum ante ipsum primis In faucibus orci luctus et Ultrices posuere cubilia cura Aenean consectetur nec',
				featureList2: 'Consectetur adipiscing elit Suspendisse at pretium tortor Vestibulum ante ipsum primis In faucibus orci luctus et Ultrices posuere cubilia cura Aenean consectetur nec',
				pricingBoxColor: '#000000',
				priceColor: '#000000',
				perMonthLabelColor: '#000000',
				buttonColor: '#000000',
				buttonTextColor: '#000000',
				featureListColor: '#000000',
				columns: '2',
				size: 'normal',
				cornerButtonRadius: '4',
			},

		} )

		expect( wrapper ).toMatchSnapshot()
	} )

	test( '3 column block edit matches snapshots', () => {
		const wrapper = edit( {
			isSelected: false,
			attributes: {
				url: 'http://www.gambit.ph/',
				url2: 'http://www.gambit.ph/',
				url3: 'http://www.gambit.ph/',
				pricingBoxTitle: 'Basic',
				pricingBoxTitle2: 'Basic',
				pricingBoxTitle3: 'Basic',
				price: '$9',
				price2: '$9',
				price3: '$9',
				perMonthLabel: 'per month',
				perMonthLabel2: 'per month',
				perMonthLabel3: 'per month',
				buttonText: 'Buy Now',
				buttonText2: 'Buy Now',
				buttonText3: 'Buy Now',
				featureList: 'Consectetur adipiscing elit Suspendisse at pretium tortor Vestibulum ante ipsum primis In faucibus orci luctus et Ultrices posuere cubilia cura Aenean consectetur nec',
				featureList2: 'Consectetur adipiscing elit Suspendisse at pretium tortor Vestibulum ante ipsum primis In faucibus orci luctus et Ultrices posuere cubilia cura Aenean consectetur nec',
				featureList3: 'Consectetur adipiscing elit Suspendisse at pretium tortor Vestibulum ante ipsum primis In faucibus orci luctus et Ultrices posuere cubilia cura Aenean consectetur nec',
				pricingBoxColor: '#000000',
				priceColor: '#000000',
				perMonthLabelColor: '#000000',
				buttonColor: '#000000',
				buttonTextColor: '#000000',
				featureListColor: '#000000',
				columns: '3',
				size: 'normal',
				cornerButtonRadius: '4',
			},

		} )

		expect( wrapper ).toMatchSnapshot()
	} )

	test( 'block save matches snapshots', () => {
		const wrapper = save( {
			isSelected: false,
			attributes: {
				url: 'http://www.gambit.ph/',
				pricingBoxTitle: 'Basic',
				price: '$9',
				perMonthLabel: 'per month',
				buttonText: 'Buy Now',
				featureList: 'Consectetur adipiscing elit Suspendisse at pretium tortor Vestibulum ante ipsum primis In faucibus orci luctus et Ultrices posuere cubilia cura Aenean consectetur nec',
				pricingBoxColor: '#000000',
				priceColor: '#000000',
				perMonthLabelColor: '#000000',
				buttonColor: '#000000',
				buttonTextColor: '#000000',
				featureListColor: '#000000',
				columns: '1',
				size: 'normal',
				cornerButtonRadius: '4',
			},
		} )

		expect( wrapper ).toMatchSnapshot()
	} )

	test( '2 column block save matches snapshots', () => {
		const wrapper = save( {
			isSelected: false,
			attributes: {
				url: 'http://www.gambit.ph/',
				url2: 'http://www.gambit.ph/',
				pricingBoxTitle: 'Basic',
				pricingBoxTitle2: 'Basic',
				price: '$9',
				price2: '$9',
				perMonthLabel: 'per month',
				perMonthLabel2: 'per month',
				buttonText: 'Buy Now',
				buttonText2: 'Buy Now',
				featureList: 'Consectetur adipiscing elit Suspendisse at pretium tortor Vestibulum ante ipsum primis In faucibus orci luctus et Ultrices posuere cubilia cura Aenean consectetur nec',
				featureList2: 'Consectetur adipiscing elit Suspendisse at pretium tortor Vestibulum ante ipsum primis In faucibus orci luctus et Ultrices posuere cubilia cura Aenean consectetur nec',
				pricingBoxColor: '#000000',
				priceColor: '#000000',
				perMonthLabelColor: '#000000',
				buttonColor: '#000000',
				buttonTextColor: '#000000',
				featureListColor: '#000000',
				columns: '2',
				size: 'normal',
				cornerButtonRadius: '4',
			},
		} )

		expect( wrapper ).toMatchSnapshot()
	} )

	test( '3 column block save matches snapshots', () => {
		const wrapper = save( {
			isSelected: false,
			attributes: {
				url: 'http://www.gambit.ph/',
				url2: 'http://www.gambit.ph/',
				url3: 'http://www.gambit.ph/',
				pricingBoxTitle: 'Basic',
				pricingBoxTitle2: 'Basic',
				pricingBoxTitle3: 'Basic',
				price: '$9',
				price2: '$9',
				price3: '$9',
				perMonthLabel: 'per month',
				perMonthLabel2: 'per month',
				perMonthLabel3: 'per month',
				buttonText: 'Buy Now',
				buttonText2: 'Buy Now',
				buttonText3: 'Buy Now',
				featureList: 'Consectetur adipiscing elit Suspendisse at pretium tortor Vestibulum ante ipsum primis In faucibus orci luctus et Ultrices posuere cubilia cura Aenean consectetur nec',
				featureList2: 'Consectetur adipiscing elit Suspendisse at pretium tortor Vestibulum ante ipsum primis In faucibus orci luctus et Ultrices posuere cubilia cura Aenean consectetur nec',
				featureList3: 'Consectetur adipiscing elit Suspendisse at pretium tortor Vestibulum ante ipsum primis In faucibus orci luctus et Ultrices posuere cubilia cura Aenean consectetur nec',
				pricingBoxColor: '#000000',
				priceColor: '#000000',
				perMonthLabelColor: '#000000',
				buttonColor: '#000000',
				buttonTextColor: '#000000',
				featureListColor: '#000000',
				columns: '3',
				size: 'normal',
				cornerButtonRadius: '4',
			},
		} )

		expect( wrapper ).toMatchSnapshot()
	} )
} )
