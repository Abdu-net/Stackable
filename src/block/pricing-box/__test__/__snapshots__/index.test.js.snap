// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pricing Box block edit matches snapshots 1`] = `
<Fragment>
  <InspectorControls>
    <PanelBody>
      <SelectControl
        label="Column Number"
        onChange={[Function]}
        options={
          Array [
            Object {
              "label": "One Column",
              "value": "1",
            },
            Object {
              "label": "Two Column",
              "value": "2",
            },
            Object {
              "label": "Three Column",
              "value": "3",
            },
          ]
        }
        value="1"
      />
      <RangeControl
        label="Corner Radius"
        max="50"
        min="1"
        onChange={[Function]}
        value="4"
      />
      <SelectControl
        label="Button Size"
        onChange={[Function]}
        options={
          Array [
            Object {
              "label": "Small",
              "value": "small",
            },
            Object {
              "label": "Normal",
              "value": "normal",
            },
            Object {
              "label": "Medium",
              "value": "medium",
            },
            Object {
              "label": "Large",
              "value": "large",
            },
          ]
        }
        value="normal"
      />
    </PanelBody>
    <PanelColorSettings
      colorSettings={
        Array [
          Object {
            "label": "Pricing Title Color",
            "onChange": [Function],
            "value": "#000000",
          },
          Object {
            "label": "Price Color",
            "onChange": [Function],
            "value": "#000000",
          },
          Object {
            "label": "Per Month Label Color",
            "onChange": [Function],
            "value": "#000000",
          },
          Object {
            "label": "Feature List Color",
            "onChange": [Function],
            "value": "#000000",
          },
        ]
      }
      initialOpen={false}
      title="Text Colors"
    />
    <PanelColorSettings
      colorSettings={
        Array [
          Object {
            "label": "Button Color",
            "onChange": [Function],
            "value": "#000000",
          },
          Object {
            "label": "Button Text Color",
            "onChange": [Function],
            "value": "#000000",
          },
        ]
      }
      initialOpen={false}
      title="Button Colors"
    />
  </InspectorControls>
  <div
    className="ugb-pricing-box column-1"
  >
    <div
      className="ugb-pricing-box-column-one"
    >
      <RichText
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="h3"
        value="Basic"
      />
      <RichText
        className="ugb-pricing-box-pricing"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="$9"
      />
      <RichText
        className="ugb-pricing-box-per-month-label"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="per month"
      />
      <ButtonEdit
        backgroundColor="#000000"
        borderRadius="4"
        color="#000000"
        onChange={[Function]}
        size="normal"
        text="Buy Now"
      />
      <RichText
        className="ugb-pricing-box-feature-list"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="Consectetur adipiscing elit Suspendisse at pretium tortor Vestibulum ante ipsum primis In faucibus orci luctus et Ultrices posuere cubilia cura Aenean consectetur nec"
      />
    </div>
    <div
      className="ugb-pricing-box-column-two"
    >
      <RichText
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="h3"
        value={undefined}
      />
      <RichText
        className="ugb-pricing-box-pricing"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value={undefined}
      />
      <RichText
        className="ugb-pricing-box-per-month-label"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value={undefined}
      />
      <ButtonEdit
        backgroundColor="#000000"
        borderRadius="4"
        color="#000000"
        onChange={[Function]}
        size="normal"
        text={undefined}
      />
      <RichText
        className="ugb-pricing-box-feature-list"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value={undefined}
      />
    </div>
    <div
      className="ugb-pricing-box-column-three"
    >
      <RichText
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="h3"
        value={undefined}
      />
      <RichText
        className="ugb-pricing-box-pricing"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value={undefined}
      />
      <RichText
        className="ugb-pricing-box-per-month-label"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value={undefined}
      />
      <ButtonEdit
        backgroundColor="#000000"
        borderRadius="4"
        color="#000000"
        onChange={[Function]}
        size="normal"
        text={undefined}
      />
      <RichText
        className="ugb-pricing-box-feature-list"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value={undefined}
      />
    </div>
  </div>
</Fragment>
`;

exports[`Pricing Box block edit matches snapshots 2`] = `
<Fragment>
  <InspectorControls>
    <PanelBody>
      <SelectControl
        label="Column Number"
        onChange={[Function]}
        options={
          Array [
            Object {
              "label": "One Column",
              "value": "1",
            },
            Object {
              "label": "Two Column",
              "value": "2",
            },
            Object {
              "label": "Three Column",
              "value": "3",
            },
          ]
        }
        value="2"
      />
      <RangeControl
        label="Corner Radius"
        max="50"
        min="1"
        onChange={[Function]}
        value="4"
      />
      <SelectControl
        label="Button Size"
        onChange={[Function]}
        options={
          Array [
            Object {
              "label": "Small",
              "value": "small",
            },
            Object {
              "label": "Normal",
              "value": "normal",
            },
            Object {
              "label": "Medium",
              "value": "medium",
            },
            Object {
              "label": "Large",
              "value": "large",
            },
          ]
        }
        value="normal"
      />
    </PanelBody>
    <PanelColorSettings
      colorSettings={
        Array [
          Object {
            "label": "Pricing Title Color",
            "onChange": [Function],
            "value": "#000000",
          },
          Object {
            "label": "Price Color",
            "onChange": [Function],
            "value": "#000000",
          },
          Object {
            "label": "Per Month Label Color",
            "onChange": [Function],
            "value": "#000000",
          },
          Object {
            "label": "Feature List Color",
            "onChange": [Function],
            "value": "#000000",
          },
        ]
      }
      initialOpen={false}
      title="Text Colors"
    />
    <PanelColorSettings
      colorSettings={
        Array [
          Object {
            "label": "Button Color",
            "onChange": [Function],
            "value": "#000000",
          },
          Object {
            "label": "Button Text Color",
            "onChange": [Function],
            "value": "#000000",
          },
        ]
      }
      initialOpen={false}
      title="Button Colors"
    />
  </InspectorControls>
  <div
    className="ugb-pricing-box column-2"
  >
    <div
      className="ugb-pricing-box-column-one"
    >
      <RichText
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="h3"
        value="Basic"
      />
      <RichText
        className="ugb-pricing-box-pricing"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="$9"
      />
      <RichText
        className="ugb-pricing-box-per-month-label"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="per month"
      />
      <ButtonEdit
        backgroundColor="#000000"
        borderRadius="4"
        color="#000000"
        onChange={[Function]}
        size="normal"
        text="Buy Now"
      />
      <RichText
        className="ugb-pricing-box-feature-list"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="Consectetur adipiscing elit Suspendisse at pretium tortor Vestibulum ante ipsum primis In faucibus orci luctus et Ultrices posuere cubilia cura Aenean consectetur nec"
      />
    </div>
    <div
      className="ugb-pricing-box-column-two"
    >
      <RichText
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="h3"
        value="Basic"
      />
      <RichText
        className="ugb-pricing-box-pricing"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="$9"
      />
      <RichText
        className="ugb-pricing-box-per-month-label"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="per month"
      />
      <ButtonEdit
        backgroundColor="#000000"
        borderRadius="4"
        color="#000000"
        onChange={[Function]}
        size="normal"
        text="Buy Now"
      />
      <RichText
        className="ugb-pricing-box-feature-list"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="Consectetur adipiscing elit Suspendisse at pretium tortor Vestibulum ante ipsum primis In faucibus orci luctus et Ultrices posuere cubilia cura Aenean consectetur nec"
      />
    </div>
    <div
      className="ugb-pricing-box-column-three"
    >
      <RichText
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="h3"
        value={undefined}
      />
      <RichText
        className="ugb-pricing-box-pricing"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value={undefined}
      />
      <RichText
        className="ugb-pricing-box-per-month-label"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value={undefined}
      />
      <ButtonEdit
        backgroundColor="#000000"
        borderRadius="4"
        color="#000000"
        onChange={[Function]}
        size="normal"
        text={undefined}
      />
      <RichText
        className="ugb-pricing-box-feature-list"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value={undefined}
      />
    </div>
  </div>
</Fragment>
`;

exports[`Pricing Box block edit matches snapshots 3`] = `
<Fragment>
  <InspectorControls>
    <PanelBody>
      <SelectControl
        label="Column Number"
        onChange={[Function]}
        options={
          Array [
            Object {
              "label": "One Column",
              "value": "1",
            },
            Object {
              "label": "Two Column",
              "value": "2",
            },
            Object {
              "label": "Three Column",
              "value": "3",
            },
          ]
        }
        value="3"
      />
      <RangeControl
        label="Corner Radius"
        max="50"
        min="1"
        onChange={[Function]}
        value="4"
      />
      <SelectControl
        label="Button Size"
        onChange={[Function]}
        options={
          Array [
            Object {
              "label": "Small",
              "value": "small",
            },
            Object {
              "label": "Normal",
              "value": "normal",
            },
            Object {
              "label": "Medium",
              "value": "medium",
            },
            Object {
              "label": "Large",
              "value": "large",
            },
          ]
        }
        value="normal"
      />
    </PanelBody>
    <PanelColorSettings
      colorSettings={
        Array [
          Object {
            "label": "Pricing Title Color",
            "onChange": [Function],
            "value": "#000000",
          },
          Object {
            "label": "Price Color",
            "onChange": [Function],
            "value": "#000000",
          },
          Object {
            "label": "Per Month Label Color",
            "onChange": [Function],
            "value": "#000000",
          },
          Object {
            "label": "Feature List Color",
            "onChange": [Function],
            "value": "#000000",
          },
        ]
      }
      initialOpen={false}
      title="Text Colors"
    />
    <PanelColorSettings
      colorSettings={
        Array [
          Object {
            "label": "Button Color",
            "onChange": [Function],
            "value": "#000000",
          },
          Object {
            "label": "Button Text Color",
            "onChange": [Function],
            "value": "#000000",
          },
        ]
      }
      initialOpen={false}
      title="Button Colors"
    />
  </InspectorControls>
  <div
    className="ugb-pricing-box column-3"
  >
    <div
      className="ugb-pricing-box-column-one"
    >
      <RichText
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="h3"
        value="Basic"
      />
      <RichText
        className="ugb-pricing-box-pricing"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="$9"
      />
      <RichText
        className="ugb-pricing-box-per-month-label"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="per month"
      />
      <ButtonEdit
        backgroundColor="#000000"
        borderRadius="4"
        color="#000000"
        onChange={[Function]}
        size="normal"
        text="Buy Now"
      />
      <RichText
        className="ugb-pricing-box-feature-list"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="Consectetur adipiscing elit Suspendisse at pretium tortor Vestibulum ante ipsum primis In faucibus orci luctus et Ultrices posuere cubilia cura Aenean consectetur nec"
      />
    </div>
    <div
      className="ugb-pricing-box-column-two"
    >
      <RichText
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="h3"
        value="Basic"
      />
      <RichText
        className="ugb-pricing-box-pricing"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="$9"
      />
      <RichText
        className="ugb-pricing-box-per-month-label"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="per month"
      />
      <ButtonEdit
        backgroundColor="#000000"
        borderRadius="4"
        color="#000000"
        onChange={[Function]}
        size="normal"
        text="Buy Now"
      />
      <RichText
        className="ugb-pricing-box-feature-list"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="Consectetur adipiscing elit Suspendisse at pretium tortor Vestibulum ante ipsum primis In faucibus orci luctus et Ultrices posuere cubilia cura Aenean consectetur nec"
      />
    </div>
    <div
      className="ugb-pricing-box-column-three"
    >
      <RichText
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="h3"
        value="Basic"
      />
      <RichText
        className="ugb-pricing-box-pricing"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="$9"
      />
      <RichText
        className="ugb-pricing-box-per-month-label"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="per month"
      />
      <ButtonEdit
        backgroundColor="#000000"
        borderRadius="4"
        color="#000000"
        onChange={[Function]}
        size="normal"
        text="Buy Now"
      />
      <RichText
        className="ugb-pricing-box-feature-list"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="Consectetur adipiscing elit Suspendisse at pretium tortor Vestibulum ante ipsum primis In faucibus orci luctus et Ultrices posuere cubilia cura Aenean consectetur nec"
      />
    </div>
  </div>
</Fragment>
`;

exports[`Pricing Box block save matches snapshots 1`] = `
<div
  className="ugb-pricing-box column-1"
>
  <div
    className="ugb-pricing-box-column-one"
  >
    <ButtonSave
      backgroundColor="#000000"
      borderRadius="4"
      color="#000000"
      size="normal"
      text="Buy Now"
      url="http://www.gambit.ph/"
    />
  </div>
</div>
`;

exports[`Pricing Box block save matches snapshots 2`] = `
<div
  className="ugb-pricing-box column-2"
>
  <div
    className="ugb-pricing-box-column-one"
  >
    <ButtonSave
      backgroundColor="#000000"
      borderRadius="4"
      color="#000000"
      size="normal"
      text="Buy Now"
      url="http://www.gambit.ph/"
    />
  </div>
  <div
    className="ugb-pricing-box-column-two"
  >
    <ButtonSave
      backgroundColor="#000000"
      borderRadius="4"
      color="#000000"
      size="normal"
      text="Buy Now"
      url="http://www.gambit.ph/"
    />
  </div>
</div>
`;

exports[`Pricing Box block save matches snapshots 3`] = `
<div
  className="ugb-pricing-box column-3"
>
  <div
    className="ugb-pricing-box-column-one"
  >
    <ButtonSave
      backgroundColor="#000000"
      borderRadius="4"
      color="#000000"
      size="normal"
      text="Buy Now"
      url="http://www.gambit.ph/"
    />
  </div>
  <div
    className="ugb-pricing-box-column-two"
  >
    <ButtonSave
      backgroundColor="#000000"
      borderRadius="4"
      color="#000000"
      size="normal"
      text="Buy Now"
      url="http://www.gambit.ph/"
    />
  </div>
  <div
    className="ugb-pricing-box-column-three"
  >
    <ButtonSave
      backgroundColor="#000000"
      borderRadius="4"
      color="#000000"
      size="normal"
      text="Buy Now"
      url="http://www.gambit.ph/"
    />
  </div>
</div>
`;
